python/paddle/v2/framework/tests/test_program.py

```diff
@@ -31,6 +33,21 @@ def test_program(self):
         self.assertEqual(1, b.idx)
         self.assertEqual(0, b.parent_idx)
 
+    def test_backward(self):
+        prog = core.ProgramDesc.__create_program_desc__()
+        self.assertIsNotNone(prog)
+        block = prog.block(0)
+        self.assertIsNotNone(block)
+
+        sum_op_desc = block.append_op()
+        sum_op_desc.set_type("sum")
+        sum_op_desc.set_input("X", ["x1", "x2"])
+        sum_op_desc.set_output("Out", ["out"])
+
+        self.assertEqual(len(block.all_ops()), 1)
+        prog.backward(set())
+        self.assertEqual(len(block.all_ops()), 3)
```

Comment:
Add more check here to make sure additional ops are exactly what we want.

Reply:
ok

Reply:
done